### PR TITLE
Add posting date extractor

### PR DESCRIPTION
**Pull request by ZwEin27**

### CODE REVIEW

File: digPostingDateExtractor/date_manipulation.py [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+        "marts": 3,
+        # "april": 4,
+        "maj": 5,
+        # "juni": 6,
+        # "juli": 7,
+        # "august": 8,
+        # "september": 9,
+        # "oktober": 10,
+        # "november": 11,
+        # "december": 12
```
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py:145]: Any idea why some of these duplicated and commented out?

File: digPostingDateExtractor/date_manipulation.py [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+        return ''
+
+
+    @staticmethod
+    def posttime_date(str, default_time, format='time'):
+        """
+        default_time can be None
+        """
+        # print "default_time ",
+        # print default_time
```
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py]: Remove commented out code

File: digPostingDateExtractor/date_manipulation.py [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+            (m, d, y) = tuples[0]
+            return DataManipulation.make_iso(y, m, d, format)
+
+        last_ditch_parse = DataManipulation.iso8601date(str, format)
+        if last_ditch_parse:
+            if format=='time':
+                return last_ditch_parse
+            else:
+                reparse_datetime = datetime.datetime.strptime(last_ditch_parse, "%Y-%m-%dT%H:%M:%S")
+                # reparse_datetime = dateutil.parser.parse(last_ditch_parse)
```
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py]: Remove commented out code

File: digPostingDateExtractor/date_manipulation.py [hunk 4]
```diff
@@ -0,0 +1,568 @@
...
+        output format: iso8601"""
+        date = date.strip()
+
+        if date_format:
+            try:
+                return datetime.strptime(date, date_format).isoformat()
+            except Exception:
+                pass
+
+        try:
```
Comment (jasonslepicka) [inline digPostingDateExtractor/date_manipulation.py:399]: I feel like there should be a better way to handle parsing dates than chaining a bunch of exceptions.  Not necessary to address this right now, but I'll open an issue after merging the PR.

**Review by jasonslepicka — APPROVED** [review]

**Comment by jasonslepicka** [comment]
@ZwEin27 Can you just remove some of the commented out code?  I'll merge it after that.

**Comment by ZwEin27** [comment]
Yeah, sure, I will do that for comments, but for codes, I think we need to talk with Amandeep first, because this is his implementation,:)

**Pull request merged**